source/feature-manager.tsx

```diff
@@ -301,7 +301,7 @@ const unload = (featureUrl: string): void => {
 	}
 };
 
-document.addEventListener('turbo:visit', () => {
+document.addEventListener('turbo:render', () => {
```

Comment:
This is the same event used to run features and it's currently attached/run _before_ our "on load" listeners. I think this is helping avoid any flashes due to momentarily-missing classes. For example if a feature removes a class on unload, the the "on load" listener will restore it immediately after, without a render/display of the page.